action.yml

```diff
@@ -27,12 +27,23 @@ inputs:
     description: Configures the branches that Semantic Release will release to.
     required: false
 outputs:
-  version:
-    description: Released version in format of X.Y.Z (major.minor.patch).
+  major:
+    description: Major component of the released version.
+  build:
+    description: Build component of the released version.
+  minor:
+    description: Minor component of the released version.
+  patch:
+    description: Patch component of the released version.
```

Comment:
This is neither lexicographical nor semantic release order. Can we just run a YAML sorter on this file?